lib/web/desktop_playback.go

```diff
@@ -182,7 +183,13 @@ func (pp *playbackPlayer) streamSessionEvents(ctx context.Context, cancel contex
 					time.Sleep(time.Duration(e.DelayMilliseconds-lastDelay) * time.Millisecond)
 					lastDelay = e.DelayMilliseconds
 				}
-				if _, err := pp.ws.Write(e.Message); err != nil {
+				// Note that e.Message is a []byte, which will be marshaled as a base64 encoded string
+				// https://pkg.go.dev/encoding/json#Marshal.
+				msg, err := utils.FastMarshal(e)
+				if err != nil {
+					pp.log.WithError(err).Errorf("failed to marshal DesktopRecording event into JSON: %v", msg)
```

Comment:
Should you break or return here?

Reply:
I don't think so. There may be a bad or otherwise unrecognized message in the playback stream, but we should probably just continue streaming if we can.

Reply:
Right - I had a better look at the context, makes sense.